plugins/balloonpanel/plugin.js

```diff
@@ -320,10 +320,13 @@
 		 *
 		 * @method attach
 		 * @param {CKEDITOR.dom.element} element The element to which the panel is attached.
-		 * @param {CKEDITOR.dom.element/Boolean} [focusElement] The element to be focused after the panel
+		 * @param {Object} options Ballonpanel focus and show options
+		 * @param {CKEDITOR.dom.element/Boolean} [options.focusElement] The element to be focused after the panel
 		 * is attached. By default the `panel` property of {@link #parts} will be focused. You might specify the element
 		 * to be focused by passing any {@link CKEDITOR.dom.element} instance.
 		 * You can also prevent changing focus at all by setting it to `false`.
+		 * Balloonpanel still support old API where focusElement was second param of attach method.
+		 * @param {Boolean} [options.show] Param defines if we want to show balloonpanel after attach. By default panel will show.
```

Comment:
JSDoc/JSDuck provides a way to indicate what is  the default value of a given property. By using this you will no longer need "By default panel will show." as it will be clear what happens by default.

Here, method works as if `options.show` is set to `true` by default.